apps/client/src/components/Location/Location.js

```diff
@@ -4,7 +4,7 @@ import React, { useContext, useEffect, useState } from "react";
 import { useForm } from "react-hook-form";
 import { useHistory } from "react-router-dom";
 
-import { OLO } from "../../config";
+import { Flow, OLO } from "../../config";
```

Comment:
This file has been separated into LocationInput and LocationResult, so these changes have no effect anymore. Also I think it would be better if these `flow` changes go on a separate PR, because they have nothing to do with "Consistent, fast sttr build output".

Reply:
Changed the name of the PR to better reflect it's purpose.

I've changed several things about `flow`. It is very related to changing the topic handling, making it default to find the json matching the slug.

I'll propose the refactor in the next dev-meeting. If we think another solution would be better, I'll change the implementation.